src/blog/2023/08/UNS-article.md

```diff
@@ -0,0 +1,68 @@
+
+The Automation Pyramid is grounded in the standard ISA-95, which aligns with IEC 62264 and DIN EN 62264. It delineates the functional hierarchy within a manufacturing enterprise. Over 25 variations of the Automation Pyramid exist in academic literature, all of them fundamentally mapping to the same core concept, tracing back to the Computer-integrated manufacturing (CIM)-Pyramid of the 1970s. Although ISA-95 does not explicitly refer to a pyramid, it introduces five functional hierarchical levels often visualized as a pyramid.
+
+![Automation Pyramid ISA95](./images/UNS/ISA95.JPG)
+
+A notable critique of ISA-95 is the absence of some operational functions and hierarchical levels commonly seen in manufacturing, leading to a rigidity that limits its applicability. This inflexibility has been acknowledged in the more recent framework, the "Reference Architectural Model Industry 4.0" RAMI 4.0 (IEC PAS 63088). As a result, the authors' introduced a "Smart Grid Architecture Model" (SGAM) with three primary dimensions: Life Cycle & Value Stream (IEC 62890), Hierarchy Levels (IEC 62264 and IEC 61512), and six main layers displaying the functional architecture of the asset and the separation into physical and digital world.
+
+![RAMI4.0](./images/UNS/RAMI40.jpeg)
+
+However, my primary critique revolves around another issue – the structure and proposed communication methodology. Models based on layers, where each tier represents a functional area and could be covered by one or more applications, almost always lead to three fundamental problems:
+
+### Problem 1: Information Loss and Transaction Costs
+
+In the traditional model, data collection flows upward from Levels 0 to 4, while planning goes downward from Level 4 to 0. Information traversing from Level 0 to Level 4 has to pass through at least four stages. Despite theoretical lossless transmission of information, the practical scenario inevitably results in some degree of information loss between levels. The upshot is that original information from Level 0 arrives at Level 4 late, altered, or not at all.
+
+### Problem 2: The Expense of One-to-One Connections 
+
+The Automation Pyramid does not explicitly provide a data management approach. Consequently, one-to-one connections between IT systems become a necessity for data transfer between levels. For example, Level 3 IT systems need at least two connections to the adjacent levels. This can lead to thousands of one-to-one interfaces between IT systems, incurring exorbitant costs for projects and maintenance.
+
+### Problem 3: AI's Dependence on Data 
+
+Artificial intelligence (AI) requires extensive, well-organized data. Given the current architecture, data would have to be manually collected and prepared from individual systems and levels. This would invariably lead to numerous new one-to-one connections, offering no flexibility. Hence, AI and the Automation Pyramid can only collaborate in a significantly restricted manner.
```

Comment:
I don't understand your point for this last problem?  Why is data need to be manually collected?